demos/risk/InfrastructureRisk/RoadNetwork/nonfunctional.csv

```diff
@@ -0,0 +1,8 @@
+﻿taxonomy,consequence,loss_type,minor,major
```

Comment:
@asthapoudel did you want to change the name of the consequence here from "collapsed" to something else?

Reply:
@raoanirudh, does the engine currently allow it?